config/config/sc-config.yaml

```diff
@@ -175,31 +215,74 @@ elasticsearch:
       limits:
         memory: 1024Mi
         cpu: 1
-  storageClass: local-storage
-  retention:
-    kubeAuditSize: 50
-    kubeAuditAge: 30
-    kubernetesSize: 50
-    kubernetesAge: 50
-    otherSize: 10
-    otherAge: 30
+    affinity:
+      podAntiAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution:
+        - weight: 1
+          podAffinityTerm:
+            topologyKey: "kubernetes.io/hostname"
+            labelSelector:
+              matchLabels:
+                role: client
+    tolerations: []
+    nodeSelector: {}
+  # Config for https://www.elastic.co/guide/en/elasticsearch/client/curator/5.8/about.html
+  curator:
+    enabled: true
+    resources:
+      limits:
+        cpu: 100m
+        memory: 128Mi
+      requests:
+        cpu: 10m
+        memory: 32Mi
+    affinity: {}
+    nodeSelector: {}
+    tolerations: []
+    retention:
+      kubeAuditSize: 50
+      kubeAuditAge: 30
+      kubernetesSize: 50
+      kubernetesAge: 50
+      otherSize: 10
+      otherAge: 30
```

Comment:
These are almost duplicated in the prod flavor, but `kubernetesAge` and `otherSize` is different. Is it intentional?

Reply:
Don't know. These are still the same values as before https://github.com/elastisys/compliantkubernetes-apps/pull/92/files#diff-e68b7d09df058f7659b32667a28cfd01c33c2b7b587012502110e1a98bc973f2L51-L57

Reply:
Ah I see! They must be the defaults we had before the flavors. Could probably remove or change them to match one of the flavors at some point, but no need to do it as part of this.